src/components/navbar/index.js

```diff
@@ -1,53 +1,173 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import Helmet from 'react-helmet';
 import Link from 'gatsby-link';
 import logo from '../../img/logo_oms_hoved.png';
+import menuGraph from '../../img/menu_graph.png';
 import styles from './navbar.module.scss';
 
-const changeLinkLanguage = language => {
-  const links = { about: 'About us', work: 'Work', products: 'Products' };
+export default class Navbar extends Component {
+  static propTypes = {
+    language: PropTypes.string,
+    location: PropTypes.shape({ pathname: {} }),
+  };
 
-  if (language === 'no') {
-    links.about = 'Om oss';
-    links.work = 'Jobb';
-    links.products = 'Produkter';
-  }
+  static defaultProps = {
+    language: 'no',
+  };
+
+  state = {
+    navOpen: false,
+  };
+
+  changePageLanguage = () => {
+    const pathSplitArray = this.props.location.pathname.split('/');
+    let returnPath;
 
-  return links;
-};
-
-const Navbar = ({ language }) => {
-  const base = `/${language}`;
-  const link = changeLinkLanguage(language);
-
-  return (
-    <nav className={styles.navbar}>
-      <ul>
-        <li>
-          <Link to={base}>
-            <img src={logo} alt="Oms logo" />
-          </Link>
-        </li>
-        <li>
-          <Link to={`${base}/products`}>{link.products}</Link>
-        </li>
-        <li>
-          <Link to={`${base}/career`}>{link.work}</Link>
-        </li>
-        <li>
-          <Link to={`${base}/about`}>{link.about}</Link>
-        </li>
-      </ul>
-    </nav>
-  );
-};
-
-Navbar.propTypes = {
-  language: PropTypes.string,
-};
-
-Navbar.defaultProps = {
-  language: 'no',
-};
-
-export default Navbar;
+    if (this.props.language === 'en') {
+      if (pathSplitArray.length < 3) {
+        returnPath = 'no';
+      } else {
+        returnPath = `/no/${pathSplitArray[2]}`;
+      }
+    } else if (pathSplitArray.length < 3) {
+      returnPath = 'en';
+    } else {
+      returnPath = `/en/${pathSplitArray[2]}`;
+    }
+
+    return returnPath;
+  };
+
+  toggleNav = () => {
+    this.setState(prevState => ({
+      navOpen: !prevState.navOpen,
+    }));
+  };
+
+  closeNav = () => {
+    this.setState({
+      navOpen: false,
+    });
+  };
+
+  render() {
+    return (
+      <header className={styles.navbar}>
+        <Helmet>
+          <body className={this.state.navOpen ? styles.noScroll : null} />
+        </Helmet>
+        <Link
+          className={`${styles.noHover} ${styles.logo}`}
+          to={`/${this.props.language}`}
+          onClick={this.closeNav}
+        >
+          <img src={logo} alt="Oms logo" />{' '}
+        </Link>
+        <div
+          className={styles.navToggle}
+          role="button"
+          tabIndex={0}
+          onClick={this.toggleNav}
+          onKeyUp={this.toggleNav}
```

Comment:
Legg gjere til en aria label for skjermlesere. F.eks 
`aria-label={this.props.language === 'en' ? 'Toggle menu' : 'Åpne/lukke meny'}`

Reply:
Istedefor <span className={styles.srOnly}>Toggle nav-menu</span> på neste linje?